tests/integration/update.bats

```diff
@@ -312,3 +312,42 @@ EOF
     check_cgroup_value "cpu.rt_period_us" 900001
     check_cgroup_value "cpu.rt_runtime_us" 600001
 }
+
+@test "update devices [minimal transition rules]" {
+	# Run a basic shell script that tries to write to /dev/null. If "runc
+	# update" makes use of minimal transition rules, updates should not cause
+	# writes to fail at any point.
+	jq '.process.args = ["sh", "-c", "while true; do echo >/dev/null; done"]' config.json > config.json.tmp
+	mv config.json{.tmp,}
+
+	# Set up a temporary console socket and recvtty so we can get the stdio.
+	TMP_RECVTTY_DIR="$(mktemp -d "$BATS_TMPDIR/runc-tmp-recvtty.XXXXXX")"
+	TMP_RECVTTY_PID="$TMP_RECVTTY_DIR/recvtty.pid"
+	TMP_CONSOLE_SOCKET="$TMP_RECVTTY_DIR/console.sock"
+	CONTAINER_OUTPUT="$TMP_RECVTTY_DIR/output"
+	("$RECVTTY" --no-stdin --pid-file "$TMP_RECVTTY_PID" \
+				--mode single "$TMP_CONSOLE_SOCKET" &>"$CONTAINER_OUTPUT" &) &
+	sleep 0.5s
```

Comment:
maybe too short, `sleep 1s` ?

Reply:
I didn't have issues with my testing, but sure I'll bump it up.